File: Qualification/Foregone_solution.py
def find_pair(value):
    value_str = str(value)
    stringa = ""
    for j in range(0, len(value_str)):
        if value_str[j] == "4":
            stringa += "1"
        else:
            stringa += "0"

    value1 = int(stringa)
    value2 = value-value1
    response = str(value1) + " " + str(value2)
    return response


t = int(input(""))
input_value = []
for i in range(0, t):
    input_value.insert(len(input_value), int(input()))

for i in range(0, t):
    print("Case #" + str(i+1)+": "+find_pair(input_value[i]))

File: Qualification/Foregone_solution.cpp
#include <iostream>
using namespace std;

int main(){
    int number_test;
    cin>>number_test;
    string data="";
    for(int i=0;i<number_test;i++){
        cin>>data;
        string value1="";
        string value2="";
        for(int j=0;j<data.length();j++){
            if(data.at(j)=='4'){
                value1+='3';
                value2+='1';
            }
            else {
                value1+=data.at(j);
                value2+='0';
            }

        }
        cout<<"Case #"<<i+1<<": "<<value1<<" "<<value2<<endl;
    }
}
